recipe/meta.yaml

```diff
@@ -22,16 +22,26 @@ requirements:
   build:
     - toolchain
     - python
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
```

Comment:
Not really, please see this page: https://github.com/conda-forge/staged-recipes/wiki/VC-features

As you can see in the page, it says that we need them for the `build` and `run` section.